Intelipro/reja
# FEAT: Add revenue analytics to revenue page

#### What does this PR do?
It adds revenue analytics to the revenue page

#### Description of Task to be completed?

- [x] display revenue growth cards
- [x] display customer pie chart
- [x] display period  total revenue, return revenue and average revenue

#### How should this be manually tested?
- Clone this repo `git clone  [the-repo-link]`
- Checkout to this branch `ft-revenue-analytics
- Install all dependencies `npm install`
- Start the server `npm run start`
-  Log in to the application and revenue page under analytics. Scroll down to see the view

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/43441107/80065755-5135ba00-8543-11ea-8b61-c9ccfa4fc182.png)

#### Questions:
